pkg/ddl/ddl_worker.go

```diff
@@ -930,13 +935,13 @@ func (w *worker) HandleDDLJobTable(d *ddlCtx, job *model.Job) (int64, error) {
 	// If running job meets error, we will save this error in job Error
 	// and retry later if the job is not cancelled.
 	schemaVer, runJobErr = w.runDDLJob(d, t, job)
+	defer d.unlockSchemaVersion(job.ID)
```

Comment:
This should result in a long lock time, such as when `runJobErr! = nil &&errorIsRetryable (runJobErr, job)` is true, it will sleep for a while.

Reply:
e4e36256cd68ad6a46a18f75f994621db8b30926